### DISCUSSION TITLE
Complete the lyrics 🎤

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#901] [Galaxy Trivia] [answered]
```
Mmm, yeah!

________, I want to give it all to you
In the ________, there's so much I want to do
And tonight, I want to lay it at your ________
'Cause girl, I was ________ for you
And girl, you were ________ for me

I was made for ________ you, baby
You were made for ________ me
And I can't get ________ of you, baby
Can you get enough of ________?
```

**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment, marked as answer]
Mmm, yeah!

**Tonight**, I want to give it all to you
In the **darkness**, there's so much I want to do
And tonight, I want to lay it at your **feet**
'Cause girl, I was **made** for you
And girl, you were **made** for me

I was made for **lovin'** you, baby
You were made for **lovin'** me
And I can't get **enough** of you, baby
Can you get enough of **me**?

The song "I Was Made for Lovin' You" was **written by Paul Stanley, Desmond Child, and Vini Poncia**. It was performed by the American rock band KISS and released as a single in 1979. The song became a commercial success and is known for its disco-influenced sound, which was a departure from KISS's typical hard rock style.
